js/ui/codeView.js

```diff
@@ -366,6 +366,15 @@ const SessionDestroyEvent = {
     SESSION_DESTROY_TOOLBOX_DESTROYED: 1,
 };
 
+var InsideActorConstraint = new Lang.Class({
+    Name: 'insideActorConstraint',
+    Extends: Clutter.Constraint,
+
+    vfunc_update_allocation: function(actor, actorBox) {
```

Comment:
Is there not a stock `ClutterConstraint` subclass that we could use instead of rolling our own?

Reply:
I've done this with the `ClutterPathConstraint` instead of a custom one.